packages/apps-config/src/extensions/aleph-zero-signer.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Known } from './types.js';
+
+import { extensionsAlephZeroSignerSVG } from '../ui/logos/extensions/index.js';
+
+export const AlephZeroSigner: Known = {
+  all: {
+    chrome: 'https://chrome.google.com/webstore/detail/aleph-zero-signer/opbinaebpmphpefcimknblieddamhmol',
+    firefox: 'https://addons.mozilla.org/en-US/firefox/addon/aleph-zero-signer/'
+  },
+  desc: 'The easiest way to get into using Aleph Zero',
+  name: 'The Aleph Zero Signer',
+  ui: {
+    logo: extensionsAlephZeroSignerSVG
+  }
+};
```

Comment:
satisfies?

Reply:
That's copied, but I'll happily do that in both the places.